Bookish/Bookish.DataAccess/AddBook.cs

```diff
@@ -11,7 +11,7 @@ public class AddBook
         public static string AddToBooks(string title, string genre, int numberOfCopies, string isbn)
         {
             int BookID = SqlReference.Library().Count + 1;
-            int AuthorID = SqlReference.Library().Count + 1;
+            int AuthorID = 1;
```

Comment:
It's worth making sure that there's a ticket on trello to replace this with some code to actually get the right author ID, and/or a comment here saying "// TODO: [blah]" so it's clear this is temporary :)